### ISSUE TITLE
change "up for grabs" to "up-for-grabs"

### ISSUE THREAD

**Issue by MattWhilden** [dotnet/corefx#805]
This appears to be the more common usage in my spot checking. Roslyn and CoreCLR have the dash version of the tag.

Consistency will make it so that sites like this: http://up-for-grabs.net/#/tags/.net%2Cc%23 work correctly.

**Comment by ellismg** [comment]
Since I was the one that changed it, I will explain my rationale.

Previously, we had the hyphens, but when I redid all of the tags, I cleaned up all of the hyphens because this looked much nicer to me when I looked at our issues.

As far as I can tell, Roslyn uses ["Up for Grabs"](https://github.com/dotnet/roslyn/labels/Up%20for%20Grabs) with spaces.

We can submit a PR to up-for-grabs.net and use whatever spelling of the tag we want.  I would prefer not to introduce hyphens.
